server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/AnalyzeVisitor.java

```diff
@@ -253,12 +253,11 @@ public Analysis visitQuery(QueryStatement queryStatement, MPPQueryContext contex
                           false)
                       .stream()
                       .collect(Collectors.toSet()));
+            } catch (MeasurementNotExistException mne) {
+              logger.warn(mne.getMessage());
+              measurementNotExistDevices.add(device);
+              continue;
             } catch (SemanticException e) {
-              if (e instanceof MeasurementNotExistException) {
-                logger.warn(e.getMessage());
-                measurementNotExistDevices.add(device);
-                continue;
-              }
               throw e;
```

Comment:
after the change,  catch SemanticException and `throw e` can be removed, right? As by default, the exception will be thrown.